### PR TITLE
Fixed the ODST Vendor (added engineering armor, and gloves instead of armguards) and a description line in the ODST examine info.

### PR DESCRIPTION
**Pull request by Flaksim**
List fix and lore info fix.

### CODE REVIEW

File: config/example/admins.txt [hunk 1]
```diff
@@ -5,4 +5,4 @@
 # Ranks can be anything defined in admin_ranks.txt           ~Carn   #
 ######################################################################
 
-# not_a_user - Admin
```
Comment (DoctorOctopusTheOctopi) [inline config/example/admins.txt:8]: File is not used on our live and should be removed from the PR.

File: maps/crashsite_zeta/backup/crashsite-zeta.dmm [hunk 2]
```diff
@@ -0,0 +1,372 @@
+"aa" = (/obj/effect/landmark/flood_spawn,/turf/unsimulated/floor/desert,/area/planet/daynight)
```
Comment (DoctorOctopusTheOctopi) [inline maps/crashsite_zeta/backup/crashsite-zeta.dmm:1]: Changes on crashsite-zeta? Did you accidentally upload this file?

File: maps/crashsite_zeta/crashsite-zeta.dmm [hunk 3]
```diff
@@ -1,172 +1,171 @@
-"aa" = (/obj/effect/landmark/flood_spawn,/turf/unsimulated/floor/desert,/area/planet/daynight)
```
Comment (DoctorOctopusTheOctopi) [inline maps/crashsite_zeta/crashsite-zeta.dmm:1]: More changes on Zeta, it thinks you've edited both the backup and the main file.

File: maps/desert_outpost/backup/desert_outpost.dmm [hunk 4]
```diff
@@ -0,0 +1,312 @@
+"aa" = (/obj/effect/landmark/flood_spawn,/turf/unsimulated/floor/desert,/area/planet/daynight)
```
Comment (DoctorOctopusTheOctopi) [inline maps/desert_outpost/backup/desert_outpost.dmm:1]: It thinks you've changed desert_outpost and its backup or uploaded a new version

File: maps/desert_outpost/desert_outpost.dmm [hunk 5]
```diff
@@ -1,57 +1,55 @@
-"aa" = (/obj/effect/landmark/flood_spawn,/turf/unsimulated/floor/desert,/area/planet/daynight)
```
Comment (DoctorOctopusTheOctopi) [inline maps/desert_outpost/desert_outpost.dmm:1]: It thinks you've changed desert_outpost and its backup or uploaded a new version

File: maps/geminus_city/New_Colony.dmm [hunk 6]
```diff
@@ -0,0 +1,1346 @@
+"aa" = (/turf/unsimulated/floor/rock2,/area/exoplanet/desert{requires_power = 0})
```
Comment (DoctorOctopusTheOctopi) [inline maps/geminus_city/New_Colony.dmm:1]: Added a new map, I believe this is your rural colony map. I don't know if its intentional or not.

File: maps/old/backup/backup/tranquility-z1.dmm [hunk 7]
```diff
@@ -0,0 +1,213 @@
+"aa" = (/turf/unsimulated/wall{tag = "icon-rock-dark"; icon_state = "rock-dark"},/area/tranquility/walls4)
```
Comment (DoctorOctopusTheOctopi) [inline maps/old/backup/backup/tranquility-z1.dmm:1]: Backup of Tranquility edited, or it thinks its been edited.

File: maps/old/backup/tranquility-z1.dmm [hunk 8]
```diff
@@ -1,213 +1,176 @@
-"aa" = (/turf/unsimulated/wall{tag = "icon-rock-dark"; icon_state = "rock-dark"},/area/tranquility/walls4)
```
Comment (DoctorOctopusTheOctopi) [inline maps/old/backup/tranquility-z1.dmm:1]: Backup of Tranquility edited, or it thinks its been edited.

File: tools/GenerateGlobalVarAccess/generateGlobalVars.bat [hunk 9]
```diff
@@ -1,3 +0,0 @@
-@echo off
```
Comment (DoctorOctopusTheOctopi) [inline tools/GenerateGlobalVarAccess/generateGlobalVars.bat:1]: Removed much needed code from the globals, its best just not to touch globals at all. This file alone makes the PR a DNM

**Review by DoctorOctopusTheOctopi — CHANGES_REQUESTED** [review]

**Comment by X0-11** [comment]
Closed as per discord discussion

**Pull request closed without merging**